src/webhook.test.ts

```
@@ -0,0 +1,134 @@
+import logger from '@/utils/logger.js';
```

ESLint

[error] 'logger' is defined but never used.

(@typescript-eslint/no-unused-vars)